golang/cosmos/app/app.go

```diff
@@ -201,14 +204,13 @@ type GaiaApp struct {
 func init() {
 	userHomeDir, err := os.UserHomeDir()
 	if err != nil {
-		panic(err)
+		stdlog.Println("Failed to get home dir %2", err)
```

Comment:
it's safe to continue without `userHomeDir`?

Reply:
In that case, `userHomeDir == ""`.  I'm not sure if it's safe, but it will at least not crash.

Reply:
oh yeah... I forget that the zero string is "" and not NULL.